src/SolverBenchmark.jl

```diff
@@ -6,6 +6,12 @@ using Printf
 # dependencies imports
 using DataFrames
 
+const formats = Dict{DataType, String}(Signed => "%5d",
+                                       AbstractFloat => "%7.1e",
```

Comment:
This should probably be `%8.1e` by default to accomodate negative numbers. We might have to change it in the LaTeX tables too.